docker/serverREADME.md

```diff
@@ -73,3 +73,67 @@ Valid commands for dolt are
 ```
 
 Learn more about Dolt use cases, configuration and guides to use dolt on our [documentation page](https://docs.dolthub.com/introduction/what-is-dolt).
+
+# How to use this image
+
+This image is for Dolt SQL Server, which is similar to MySQL Docker Image. Running this image without any arguments 
+is equivalent to running `dolt sql-server --host 0.0.0.0 --port 3306` command locally. The reason for persisted host
+and port is that it allows user to connect to the server inside the container from the local host system through
+port-mapping.
+
+To check out supported options for `dolt sql-server`, you can run the image with `--help` flag.
+
+```shell
+$ docker run dolthub/dolt-sql-server:latest --help
+```
+
+### Connect to the server in the container from the host system
+
+To be able to connect to the server running in the container, we need to set up a port to connect to locally that
+maps to the port in the container. The host is set to `0.0.0.0` for accepting connections to any available network 
+interface.
+
+```shell
+$ docker run -p 3307:3306 dolthub/dolt-sql-server:latest
+```
+
+Now, you have running server in the container, and we can connect it by specifying any host, port as 3307 and user 
+as root since it is set to default because we did not define configurations when running the server.
```

Comment:
```suggestion
Now, you have a running server in the container, and we can connect to it by specifying our host, 3307 for the port, and root for the user, 
since that's the default user and we didn't provide any configuration when running the server.
```